apps/common-app/src/examples/RuntimeTests/tests/animations/withTiming/objects.test.tsx

```diff
@@ -97,6 +87,6 @@ const styles = StyleSheet.create({
   box: {
     height: 100,
     width: 100,
-    margin: 30,
+    margin: 0,
```

Comment:
Any margin may affect the position.

Reply:
But why this is a problem here? 🤔

Reply:
Because of testing the values of "left", "top", etc.